google-api-client/src/main/java/com/google/api/client/googleapis/testing/json/GoogleJsonResponseExceptionFactoryTesting.java

```diff
@@ -59,7 +59,10 @@ public static GoogleJsonResponseException newMock(JsonFactory jsonFactory,
     MockLowLevelHttpResponse otherServiceUnavaiableLowLevelResponse =
         new MockLowLevelHttpResponse()
         .setStatusCode(httpCode)
-        .setReasonPhrase(reasonPhrase);
+        .setReasonPhrase(reasonPhrase)
+        .setContentType(Json.MEDIA_TYPE)
```

Comment:
Looks like we're missing an import here or a constant definition.

Reply:
Whoops, fixed.